bigtop-packages/src/common/kafka/do-component-build

```diff
@@ -27,8 +27,6 @@ BUILD_OPTS="-Divy.home=${HOME}/.ivy2 -Dsbt.ivy.home=${HOME}/.ivy2 -Duser.home=${
 #        http://maven.40175.n5.nabble.com/Not-finding-artifact-in-local-repo-td3727753.html
 export MAVEN_OPTS="-Xmx2g -XX:ReservedCodeCacheSize=512m"
 
-gradle wrapper
```

Comment:
To get more info - why is this command dropped?

Reply:
It just fails dut to unavailable resources. The cause might be version mismatch of gradle. (bigtop_toolchain install 5.6.2.)

Since [generated wrapper script](https://github.com/apache/kafka/blob/2.8.1/gradlew) is provided, we don't need to generate it on our own.

```
$ gradle wrapper
Starting a Gradle Daemon, 1 busy Daemon could not be reused, use --status for details

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'kafka'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not resolve org.eclipse.jgit:org.eclipse.jgit:latest.release.
     Required by:
         project : > org.ajoberstar.grgit:grgit-core:4.1.0
      > Failed to list versions for org.eclipse.jgit:org.eclipse.jgit.
         > Unable to load Maven meta-data from https://dl.bintray.com/content/netflixoss/external-gradle-plugins/org/eclipse/jgit/org.eclipse.jgit/maven-metadata.xml.
            > Could not get resource 'https://dl.bintray.com/content/netflixoss/external-gradle-plugins/org/eclipse/jgit/org.eclipse.jgit/maven-metadata.xml'.
               > Could not GET 'https://dl.bintray.com/content/netflixoss/external-gradle-plugins/org/eclipse/jgit/org.eclipse.jgit/maven-metadata.xml'. Received status code 502 from server: Bad Gateway

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 18s
```

Reply:
Ack thanks for the explanation :)